### ISSUE TITLE
If a user restricts "/**" then no Stormpath page can be accessed

### ISSUE THREAD

**Issue by mrioan** [stormpath/stormpath-sdk-java#299]
If a user does something like this:
```
   @Override
   protected void configure(HttpSecurity http) throws Exception {
        http
                .authorizeRequests()
                .antMatchers("/restricted").authenticated()
                .antMatchers("/**").denyAll();
    }
```
Then our endpoints (like /login, /register, etc) are not accessible.

**Comment by lhazlewood** [comment]
Hrm - need to find out why.  I thought our configuration occurs first in
Spring Boot startup before the users executes and so ours would be allowed.

--
Les

**Comment by mrioan** [comment]
I already have a fix for this. Spring Security has a way to force patterns
to be always available. I did so for all of our *enabled* endpoints

On Oct 3, 2015 14:55, "Les Hazlewood" <notifications@github.com> wrote:
>
> Hrm - need to find out why. I thought our configuration occurs first in
> Spring Boot startup before the users executes and so ours would be
allowed.
>
> --
> Les
>

**Comment by lhazlewood** [comment]
Nice!

--
Les
@lhazlewood

**Comment by dogeared** [comment]
Awesome!

==============================================
Micah Silverman, CSM <http://bit.ly/WikiScrum>
Phone: 631.606.8928
Co-Author: Mastering Enterprise Javabeans 3.0 <http://bit.ly/MasteringEJB3>
==============================================
"Debugging is twice as hard as writing the code in the first
place. Therefore, if you write the code as cleverly as
possible, you are, by definition, not smart enough to debug
it." by Brian Kernighan
==============================================
[image: Namez]
<http://namez.com/profiles/1720-micah-silverman/?autoPlay=true>

**Comment by mrioan** [comment]
Solved in #304
